docs/EN_US/ConfiguringHPCC/ConfiguringHPCC.xml

```diff
@@ -2417,6 +2417,29 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
 
         <?hard-pagebreak ?>
 
+        <sect3>
```

Comment:
docs/*/ConfiguringHPCC/ are BM only docs, should be left unchanged as this service will not be part of BM.